doc/sourmash-sketch.md

```diff
@@ -0,0 +1,147 @@
+
+If a field isn't specified, then the default value for that sketch type is used; so, for example, `sourmash sketch dna -p abund` would calculate a sketch with `k=31,scaled=1000,abund`. See below for the defaults.
+
+### Default parameters
+
+The default parameters for sketches are as follows:
+
+* dna: `k=31,scaled=1000,noabund`
+* protein: `k=10,scaled=200,noabund`
+* dayhoff: `k=16,scaled=200,noabund`
+* hp=`k=42,scaled=200,noabund`
+
+These were chosen by a committee of PhDs as being good defaults for an initial analysis, so, beware :).
+
+More seriously, the DNA parameters were chosen based on the analyses done by Koslicki and Falush in [MetaPalette: a k-mer Painting Approach for Metagenomic Taxonomic Profiling and Quantification of Novel Strain Variation](https://msystems.asm.org/content/1/3/e00020-16).
+
+The protein, dayhoff, and hp parameters were selected based on unpublished research results and/or magic formulas. We are working on publishing the results! Please ask on the [issue tracker](https://github.com/dib-lab/sourmash/issues) if you are curious.
+
+### More complex parameter string examples
+
+Below are some more complicated `sourmash sketch` command lines:
+
+* `sourmash sketch dna -p k=51` - default to a scaled=1000 and noabund for a k-mer size of 51 (based on moltype/command)
+* `sourmash sketch dna -p k=31,k=51,k=21` - compute multiple ksizes, using the defaults otherwise
+* `sourmash sketch translate -p k=20,num=500,protein -p k=19,num=400,dayhoff,abund -p k=30,scaled=200,hp` - compute multiple ksizes, moltypes, and scaled/num.
+
+### Locations for output files
+
+Signature files can contain multiple signatures and sketches. Use `sourmash sig describe` to get details on the contents of a file.
+
+You can use `-o <filename>` to specify a file output location for all the output signatures; `-o -` means stdout. This does not merge signatures unless `--merge` is provided.
+
+Specify `--outdir` to put all the signatures in a specific directory.
+
+### Downsampling and flattening signatures
+
+Calculating signatures is probably the most time consuming part of using sourmash, and it is the only part that requires access to the raw data. Moreover, the output signatures are generally much smaller than the input data. So, we generally suggest calculating a large set of signatures once.
+
+To support this, sourmash can do two kinds of signature conversion without going back to the raw data.
+
+First, you can downsample `num` and `scaled` signatures using `sourmash sig downsample`.  For any sketch calculated with `num` parameter, you can decrease that `num`. And, for any `scaled` parameter, you can increase the `scaled`. This will decrease the size of the sketch accordingly; for example, going from a num of 5000 to a num of 1000 will decrease the sketch size by a factor of 5, and going from a scaled of 1000 to a scaled of 10000 will decrease the sketch size by a factor of 10.
```

Comment:
```suggestion
First, you can downsample `num` and `scaled` signatures using `sourmash sig downsample`.  For any sketch created with `num` parameter, you can decrease that `num`. And, for any `scaled` parameter, you can increase the `scaled`. This will decrease the size of the sketch accordingly; for example, going from a `num` of 5000 to a `num` of 1000 will decrease the sketch size by a factor of 5, and going from a `scaled` of 1000 to a `scaled` of 10000 will decrease the sketch size by a factor of 10.
```